update-links.js

```diff
@@ -0,0 +1,76 @@
+const fs = require("fs/promises")
+const F = require("fluture")
+const path = require("path")
+const {
+  curry,
+  filter,
+  fork,
+  identity: I,
+  j2,
+  lines,
+  map,
+  pipe,
+  readFile,
+  split,
+  always: K,
+  chain,
+} = require("snang/script")
```

Comment:
Do you see this as a library for writing scripts in the frontend repositories going forward? I'm interested what some of the advantages are compared to writing the script out imperatively. Also, I notice that you wrote it yourself 😉

Reply:
Yeah, for me this tool is a very quick way of iterating over streamables; I think in general there are alternatives / trade-offs, but the intention was that you can start by doing something like:

`cat linktree | snang -P "lines | filter(I) | map(split(': '))"`

Then after a bit of play I arrived at:

```
cat linktree | snang -P "lines | filter(I) | map(split(': ')) | z => 'export const links = ' + j2(z)" -u > update-links.js
```

which was the beginning of this actual file.

I think there might be an argument for maintainability, but honestly I'm better at maintaining this kind of code than a more imperative style.

Reply:
This is an interesting pattern to generate a file - it would be really nice if there was a bit of documentation on this file and how to use it.

Short of that, I would expect anyone who has to update links.js to simply edit the file in line.